lib/v2/iwara/index.js

```diff
@@ -1,43 +1,54 @@
 const got = require('@/utils/got');
-const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
 
 const rootUrl = 'https://ecchi.iwara.tv';
```

Comment:
https://ecchi.iwara.tv/users/kelpie will be redirected to https://www.iwara.tv/users/kelpie